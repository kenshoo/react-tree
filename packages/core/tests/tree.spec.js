import React from "react";
import { shallow } from "enzyme";
import Tree from "../src";
import toJson from "enzyme-to-json";

describe("tree component", () => {
  test("default", () => {
    const cmp = shallow(<Tree />);
    expect(toJson(cmp)).toMatchSnapshot();
  });
});
