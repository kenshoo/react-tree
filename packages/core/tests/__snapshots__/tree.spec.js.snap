// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tree component default 1`] = `
<EmotionCssPropInternal
  __EMOTION_LABEL_PLEASE_DO_NOT_USE__="Tree"
  __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
  css={
    Object {
      "backgroundColor": "#F8F9FA",
      "border": "1px solid #E1E4EB",
      "borderRadius": "4px",
      "boxShadow": "0 6px 10px 0 rgba(27, 32, 70, 0.09)",
      "boxSizing": "border-box",
      "color": "#545769",
      "fontFamily": "Arial, Helvetica, sans-serif",
      "height": "280px",
      "position": "relative",
      "width": "230px",
    }
  }
>
  <Header
    getStyles={[Function]}
    onClick={[Function]}
    parents={Array []}
  />
  <Input
    getStyles={[Function]}
    searchTerm=""
    setSearchTerm={[Function]}
  />
  <EmotionCssPropInternal
    __EMOTION_LABEL_PLEASE_DO_NOT_USE__="Tree"
    __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
    css={
      Object {
        "boxSizing": "border-box",
        "maxHeight": "203px",
        "overflowY": "auto",
      }
    }
  >
    <NoResults
      getStyles={[Function]}
      text="No matching results"
    />
  </EmotionCssPropInternal>
</EmotionCssPropInternal>
`;
